aspnetcore/tutorials/min-web-api/samples/6.x/todoDTO/Program.cs

```diff
@@ -0,0 +1,105 @@
+#region snippet_all
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDatabaseDeveloperPageExceptionFilter();
+builder.Services.AddDbContext<TodoDb>(opt => opt.UseInMemoryDatabase("TodoList"));
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseDeveloperExceptionPage();
+}
+
+#region snippet_get
+app.MapGet("/", () => "Hello World!");
+
+app.MapGet("/todoitems", async (TodoDb db) =>
+    await db.Todos.Select(x => TodoItemDTO.ItemToDTO(x)).ToListAsync());
```

Comment:
This would be cleaner as:
```suggestion
    await db.Todos.Select(x => new TodoItemDTO(x)).ToListAsync());
```